syntax = "proto3";

message Order {
    int32 id = 1;
    int32 user_id = 2;
    int32 product_id = 3;
    int32 quantity = 4;
    string status = 5;
    string order_date = 6;
}

message CartItem {
    int32 id = 1;
    int32 user_id = 2;
    int32 product_id = 3;
    int32 quantity = 4;
    string added_at = 5;
    string updated_at = 6;
}

message CartItemAdded {
    CartItem cart_item = 1;
}

message CartItemUpdated {
    CartItem cart_item = 1;
}

message CartItemRemoved {
    CartItem cart_item = 1;
}
